wrangler.toml

```diff
@@ -11,3 +11,6 @@ command = "npm install && npm run build"
 [build.upload]
 format = "modules"
 main = "./shim.mjs"
+[[build.upload.rules]]
+type = "Text"
+globs = ["**/*.txt"]
```

Comment:
Isn't this implicit? Going by https://github.com/cloudflare/wrangler/issues/1841#issuecomment-818959912 at least I'd think so.